gemet/thesaurus/models.py

```diff
@@ -25,6 +25,13 @@
     'seeAlso',
 ]
 
+EDIT_URL_NAMES = {
```

Comment:
Move this to `edit_views.py` or `definitions.py`. If you choose the latter, move all constants to this new file.